apps/web/src/components/PaginatedTable/index.tsx

```diff
@@ -128,7 +129,7 @@ export const PaginatedTable: FC<PaginatedTableProps> = function ({
         emptyState={DEFAULT_TABLE_EMPTY_STATE}
       >
         <div className="flex flex-col gap-6">
-          <Filters />
+          {tableTopSlot && tableTopSlot}
```

Comment:
Why did you do this change?

Reply:
The idea is to decouple the `Filter` component from the `PaginatedTable` component. It's a common strategy when building components to use the `slot` concept: you can pass by a prop whatever element (`ReactNode`) and will be rendered in this section: `tableTop` indicates the place where it will be rendered

Reply:
I’m reverting this because having both in the same card makes it look a bit too crowded. Let’s separate them into two different cards, with the filters on top of the paginated table.